learning_resources_search/indexing_api.py

```diff
@@ -192,23 +190,15 @@ def clear_and_create_index(*, index_name=None, skip_mapping=False, object_type=N
     }
 
     if object_type == COMBINED_INDEX:
```

Comment:
imo the naming of these indexes are getting a bit confusing (constants.COMBINED_INDEX vs constants.ALL_INDEXES vs constants.BOTH_INDEXES) might be better to call this HYBRID_INDEX or HYBRID_COMBINED_INDEX etc